### DISCUSSION TITLE
有类似luckysheet里的导入导出功能或者像window.luckysheet的输出吗？

### DISCUSSION THREAD

**Discussion by fddqfddq** [dream-num/univer#2209] [Q&A | 常见问题]
就是想初始化表格，有类似luckysheet里的导入导出功能吗，想直接生成celldata的数据源用来初始化
或者有像window.luckysheet可在控制台查看的类似的吗？

**Comment by fddqfddq** [comment]
因为excel里有公式和样式单元格合并等，想直接通过将excel里的模板内容复制到univer sheet中然后用它的数据用来初始化，类似luckysheet里的api : getSheetData()

**Comment by fddqfddq** [comment]
想了一个办法，我更新了excel后，在onSelectionChange事件里去查看，可以看到数据源了。
activeWorkbook.onSelectionChange((selection) => {
  console.log(selection);
  const savedData = univerAPI.getActiveWorkbook().getSnapshot();
  console.log('after');
  console.info('savedData', savedData);
});
![image](https://github.com/dream-num/univer/assets/10627223/573fc350-31d3-4e4c-b8c7-d33445c91020)

**Comment by yooyea** [comment]
```js
        createUnit(){
            let data = localStorage.getItem('v')
            data = data ? JSON.parse(data) : this.data
            const unit = this.univer.createUnit(UniverInstanceType.UNIVER_SHEET, data);
            return unit
        },
        save() {
            this.data = this.unit.save()
            localStorage.setItem('v', JSON.stringify(this.data))
        },
```
save 方法能够保存所有配置包括数据源
